app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/KnowledgeManagementPortalArticleController.php

```diff
@@ -75,6 +75,7 @@ public function show(KnowledgeBaseCategory $category, KnowledgeBaseItem $article
                 'slug' => $category->slug,
                 'name' => $category->name,
                 'description' => $category->description,
+                'parentCategory' => $category->parentCategory,
```

Comment:
You may want to consider making this a `KnowledgeBaseCategoryData` item as well so that only the needed data is passed.